dev/com.ibm.ws.security.javaeesec_fat/fat/src/com/ibm/ws/security/javaeesec/fat/HttpAuthenticationMechanismDBNoUserTest.java

```diff
@@ -0,0 +1,38 @@
+package com.ibm.ws.security.javaeesec.fat;
+
+import org.junit.runner.RunWith;
+
+import componenttest.annotation.MinimumJavaLevel;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.impl.LibertyServerFactory;
+
+/*
```

Comment:
Wrong copyright. If you copied find where this might have come from and fix that one too.